docs/cli/edgedb_toml.rst

```diff
@@ -0,0 +1,44 @@
+.. _ref_cli_edgedb_toml:
+
+===========
+edgedb.toml
+===========
+
+The ``edgedb.toml`` file is a key component of EdgeDB's project system. If this
+file is present in a directory, it signals to the CLI that the directory is an
+instance-linked EdgeDB project. It supports two configuration settings across
+two tables:
+
+
+``[edgedb]`` table
+==================
+
+- ``server-version``- The server version of the EdgeDB project.
+
+  .. note::
+
+      The version specification is assumed to be **a minimum version**, but the
+      CLI will *not* upgrade to subsequent major versions. This means if the
+      version specified is ``3.1`` and versions 3.2 and 3.3 are available, 3.3
+      will be installed, even if version 4.0 is also available.
+
+      To specify an exact version, prepend with ``=`` like this: ``=3.1``
```

Comment:
I think we support the full range of Rust cargo-flavored semver specifiers? Do we want to point that out and link to some reference material like https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html?